packages/wonder-blocks-layout/components/media-layout.js

```diff
@@ -4,10 +4,25 @@ import type {StyleDeclaration} from "aphrodite";
 import type {StyleType} from "@khanacademy/wonder-blocks-core";
 
 import MediaLayoutContext from "./media-layout-context.js";
-import {VALID_MEDIA_SIZES} from "../util/specs.js";
 import type {MediaSize, MediaSpec} from "../util/types.js";
 import type {Context} from "./media-layout-context.js";
 
+const queries = [
```

Comment:
Hmm - I don't like this duplication of constants from the `specs.js` file. I would prefer that you do something like:

```
const queries: Array<string> = [
	...Object.values(MEDIA_DEFAULT_SPEC),
	...Object.values(MEDIA_INTERNAL_SPEC),
	...Object.values(MEDIA_MODAL_SPEC),
].map(spec => spec.query);
```

Reply:
I ended up having to do:
```
const queries = [
    ...((Object.values(MEDIA_DEFAULT_SPEC): any): Array<string>),
	...((Object.values(MEDIA_INTERNAL_SPEC): any): Array<string>),
	...((Object.values(MEDIA_MODAL_SPEC): any): Array<string>),
];
```
😞

Reply:
That wasn't right either.. but this is:
```
const queries = [
    ...Object.values(MEDIA_DEFAULT_SPEC).map((spec: any) => spec.query),
    ...Object.values(MEDIA_INTERNAL_SPEC).map((spec: any) => spec.query),
    ...Object.values(MEDIA_MODAL_SPEC).map((spec: any) => spec.query),
];
```